components/interview/interfaces/name-generator/NodePanels.tsx

```diff
@@ -9,13 +10,17 @@ type Panel = {
 };
 
 export default forwardRef(function NodePanels(
```

Comment:
Just a general reminder for the future that `forwardRef` is going to be deprecated in React 19, and removed in a future version. We can just pass ref as a prop now.